spec/integration/application_view/part_namespace_spec.rb

```diff
@@ -141,8 +144,9 @@ module CustomParts
           }
         end
 
-        it "is the matching module within the slice" do
-          is_expected.to eq TestApp::Views::CustomParts
+        # FIXME: @jodosha ask @timriley
```

Comment:
@timriley This is failing and IDK why. Could you please help me? Thanks 🙏

Reply:
Sorted per the same fix as above :)

Reply:
Thank you. 🙏